docs: update phrasing and fix code snippet (#62031)
angular/angular: adev/src/content/tutorials/first-app/steps/05-inputs/README.md

<docs-video src="https://www.youtube.com/embed/eM3zi_n7lNs?si=WvRGFSkW_7_zDIFD&amp;start=241"/>

## What you'll learn

Your app's `HousingLocation` template has a `HousingLocation` property to receive input.

## Conceptual preview of Inputs

[Inputs](api/core/input) allow components to specify data that can be passed to it from a parent component.

In this lesson, you'll define an `input` property in the `HousingLocation` component that enables you to customize the data displayed in the component.

Learn more in the [Accepting data with input properties](guide/components/inputs) and [Custom events with outputs](guide/components/outputs) guides.

<docs-workflow>

<docs-step title="Import the input() function">
In the code editor, import the `input` helper method from `@angular/core` and the `HousingLocation` component.

> <docs-code header="Import HousingLocation and Input in src/app/housing-location/housing-location.ts" path="adev/src/content/tutorials/first-app/steps/06-property-binding/src/app/housing-location/housing-location.ts" visibleLines="[1,3]"/>

Reviewer: It looks like the visible lines aren't correct. Can we take care of it in this PR ? 
<img width="675" alt="Screenshot 2025-06-13 at 15 32 43" src="https://github.com/user-attachments/assets/5e29f27e-2465-4248-82a8-1b11ba2d849c" />

</docs-step>

<docs-step title="Add the Input property">
Add a required property called `housingLocation` and initialize it using `input.required()` with the type `HousingLocationInfo`.

  <docs-code header="Import HousingLocation and Input in src/app/housing-location/housing-location.ts" path="adev/src/content/tutorials/first-app/steps/06-property-binding/src/app/housing-location/housing-location.ts" visibleLines="[11,13]"/>

> You have to add `.required` after `input` to indicate that the parent component must provide a value. In our example application, we know this value will always be passed in — this is by design. The `.required()` call ensures that the TypeScript compiler enforces this and treats the property as non-nullable when this component is used in a template.

Reviewer: nit:
```suggestion
You have to invoked the  `required` method on `input` to indicate that the parent component must provide a value. In our example application, we know this value will always be passed in — this is by design. The `.required()` call ensures that the TypeScript compiler enforces this and treats the property as non-nullable when this component is used in a template.
```
